app/scripts/presenters/visit_presenter.js

```diff
@@ -3,12 +3,20 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 define([
-  'underscore'
-], function (_) {
+  'underscore',
+  'lib/image_proxy'
+], function (_, imageProxy) {
   'use strict';
 
   var MINIMUM_IMAGE_ENTROPY = 1.75;
-  var MINIMUM_IMAGE_WIDTH = 270;
+  var IMAGE_DISPLAY_WIDTH = 270;
+  var IMAGE_DISPLAY_HEIGHT = 180;
```

Comment:
Without digging too far into the code, not sure if I'm reading the logic:
Should these be equal values?
```js
var IMAGE_DISPLAY_HEIGHT = 180;
```

vs

```css
# app/styles/modules/_visit.scss

// Size variations
&.medium {
  .image img {
    @include two-by-three(160px);
  }
}
```

Should `IMAGE_DISPLAY_HEIGHT` be 160px, or should it be `@include two-by-three(180px);` or are these two completely different things?